src/main/java/org/breedinginsight/daos/TraitDAO.java

```diff
@@ -419,7 +422,7 @@ public Trait updateTraitBrAPI(Trait trait, Program program) {
                 existingVariable.setStatus("archived");
             }
             existingVariable.putAdditionalInfoItem(TAGS_KEY, trait.getTags());
-
+            existingVariable.putAdditionalInfoItem(FULLNAME_KEY, trait.getFullName());
```

Comment:
Should this also not write the additionalInfo item if fullName is null like the create call?

Reply:
Added the precaution just in case and checked that it didn't prevent clearing the fullname on an edit.